astropy/utils/masked/function_helpers.py

```diff
@@ -587,7 +587,7 @@ def median(a, axis=None, out=None, **kwargs):
 
     a = Masked(a)
 
-    if NUMPY_LT_1_25:
+    if NUMPY_LT_1_24:
```

Comment:
Hmm... What does this mean? An upstream patch that made it into 1.24rc was backed out last minute?

Reply:
Oh, no, wait... a patch that was not in 1.24rc got backported from 1.25.dev last minute?

Reply:
Indeed, I think so. Bit weird, but I'm not exactly sure where the were in the release schedule when I made my PR.